### PR TITLE
Feat/fix authentication

### PR DESCRIPTION
**Pull request by phoenixdahdev**
# Description

**Closes #issue_number_here**

This PR addresses the issues related to authentication and makes general improvements to all authentication routes to meet the required standards.

# Changes Proposed

## What Were You Told to Do?

- Fix authentication to meet the required standard.
- General fix-up on all authentication routes.

## What Did You Do?

- Improved validation on authentication routes.
- Refactored authentication logic to enhance security and performance.
- Updated dependencies as required.
- Adjusted error handling to provide more descriptive messages.

# Check List

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR are clear and explain the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages style matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit tests.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos

---

### CODE REVIEW

File: src/app/(landing-routes)/(home)/pricing/page.tsx
Comment (gh-i19n) [inline src/app/(landing-routes)/(home)/pricing/page.tsx:1]: there is an open comment on line 49.. Buka doesnt like this..i think we should enforce same standard on the frontend..you can take out the "//" on your next PR

**Review by gh-i19n — COMMENTED** [review]
This looks good in general

**Comment by gh-i19n** [comment]
Oh concerning, your PR... writing a more constructive PR Description is important, it guides the reviewer on what to expect during reviews.

**Review by Justfemi — APPROVED** [review]

**Comment by phoenixdahdev** [comment]
> Oh concerning, your PR... writing a more constructive PR Description is important, it guides the reviewer on what to expect during reviews.

fixed sorry that was an oversight on my path

**Review by gh-i19n — APPROVED** [review]

File: src/actions/register.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
 "use server";
 
 import axios from "axios";
-import { cookies } from "next/headers";
+// import { cookies } from "next/headers";
```
Comment (billmal071) [inline src/actions/register.ts:4]: You left this and it got merged. @incredible-phoenix246 @kinxlo

**Pull request merged**